Detectors/FIT/FT0/calibration/src/FT0ChannelTimeTimeSlotContainer.cxx

```diff
@@ -63,8 +63,6 @@ void FT0ChannelTimeTimeSlotContainer::fill(const gsl::span<const FT0CalibrationI
       mHistogram[chID]->Fill(chTime);
       ++mEntriesPerChannel[chID];
       LOG(debug) << "@@@@entries " << mEntriesPerChannel[chID] << " chID " << int(chID) << " time " << chTime << " tiestamp " << uint64_t(entry.getTimeStamp());
-    } else {
-      LOG(fatal) << "Invalid channel data";
```

Comment:
is this protection against coding error or corrupted data? Normally, the latter should not lead to fatal but raise an alarm (LOG(alarm)) and pass over it.

Reply:
This is protection was done before look-up table implementation, now its not actual. Later reading of dead channel map will be add

Reply:
So, shall I close this PR?

Reply:
Please  merge it. We have problem to test calibration @P2 with laser data with addition channels for reference PMTs.